mllib/src/main/scala/org/apache/spark/ml/classification/RandomForestClassifier.scala

```diff
@@ -222,16 +222,10 @@ final class RandomForestClassificationModel private[ml] (
   /**
    * Estimate of the importance of each feature.
    *
-   * This generalizes the idea of "Gini" importance to other losses,
-   * following the explanation of Gini importance from "Random Forests" documentation
-   * by Leo Breiman and Adele Cutler, and following the implementation from scikit-learn.
+   * Each feature's importance is the average of its importance across all trees in
```

Comment:
SGTM, but this should still note that the importances are averaged over trees and that this matches scikit-learn.  I also like saying explicitly that the importances sum to 1.